src/pybind/mgr/mgr_module.py

```diff
@@ -299,6 +300,12 @@ def _parse_args(self):
                     arg_d[k] = v
                 else:
                     self.args_dict[v] = arg_d
+                if k == 'kw' and v == 'true':
+                    kwargs_switch = True
+                else:
+                    if kwargs_switch:
+                        # we saw a kwarg earlier and now a positional
+                        logging.warn('Found positional argument after kwarg')
```

Comment:
is there a reason why we'd warn about positional arguments after a k/v pair was found? I don't see a particular technical difficulty in handling them, especially since we know which arguments are expected to be k/v pairs.

Reply:
Here (while setting up the command and argument structures) I only warn since the `kw=true` currently enables optional passing as a kwarg. The user can also pass the arguments as positional arguments.